scalableminds/webknossos
# Fix DataSets update query for zero-datasets case

### Steps to test:
- remove all datasets from your binaryData directory
- dataset refresh in wK should properly show them as »no longer available on the datastore«
- no SQL Error logs should be in the console

### Issues:
- fixes #2695

------
